scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -28,7 +28,11 @@ workflow GvsAssignIds {
   String vcf_header_lines_schema_json = '[{"name":"vcf_header_lines_hash","type":"STRING","mode":"REQUIRED"}, {"name":"vcf_header_lines","type":"STRING","mode":"REQUIRED"},{"name":"is_expected_unique","type":"BOOLEAN","mode":"REQUIRED"}]'
   String sample_vcf_header_schema_json = '[{"name": "sample_id","type": "INTEGER","mode": "REQUIRED"}, {"name":"vcf_header_lines_hash","type":"STRING","mode":"REQUIRED"}]'
   String sample_load_status_schema_json = '[{"name": "sample_id","type": "INTEGER","mode": "REQUIRED"},{"name":"status","type":"STRING","mode":"REQUIRED"}, {"name":"event_timestamp","type":"TIMESTAMP","mode":"REQUIRED"}]'
-  String sample_chromosome_ploidy_schema_json = '[{"name": "sample_id","type": "INTEGER","mode": "REQUIRED"},{"name": "chromosome","type": "INTEGER","mode": "REQUIRED"},{"name": "ploidy","type": "INTEGER","mode": "REQUIRED"}]'
+  # The schema below for the `sample_chromosome_ploidy` table matches that in the AoU `echo` dataset, which differs from
+  # the standard `sample_chromosome_ploidy` schema by the addition of the `genotype` column. At the time of this writing
+  # it is not expected that any additional samples will be loaded into the `echo` dataset, but if there are additional
+  # samples then this code will match the actual `echo` schema.
+  String sample_chromosome_ploidy_schema_json = '[{"name": "sample_id","type": "INTEGER","mode": "REQUIRED"},{"name": "chromosome","type": "INTEGER","mode": "REQUIRED"},{"name":"genotype","type":"STRING","mode":"NULLABLE"},{"name": "ploidy","type": "INTEGER","mode": "REQUIRED"}]'
```

Comment:
These changes aren't actually necessary for supporting RWB extract. They likely won't hurt, but they are not necessary for this PR as RWB does not run this wdl.

Reply:
Yeah that's kinda what I was trying to say with the comment. 🙂 It was bugging me that this was out of sync with `SchemaUtils.java`, even if we're super unlikely to ever need these changes.